src/app/pages/OnDemandTvPage/OnDemandTvPage.jsx

```diff
@@ -88,7 +87,7 @@ const OnDemandTvPage = ({ pageData, mediaIsAvailable, MediaError }) => {
   } = useContext(ServiceContext);
   const { isAmp } = useContext(RequestContext);
   const location = useLocation();
-  const darkMode = useToggle('cinemaModeTV').enabled;
+  const darkMode = true;
```

Comment:
We can delete this and just pass along: 

`          <OnDemandParagraphContainer
            text={shortSynopsis}
            darkMode
          />`